WordPress/Classes/Utility/Logging/SentryStartupEvent.swift

```diff
@@ -25,13 +34,20 @@ startup time. This will block the thread. Do not use unless you're sure.
         }
         let semaphore = DispatchSemaphore(value: 0)
         let event = Event(level: .debug)
-        let lastError = errors.removeLast()
-        event.message = "\(title): \(lastError)"
-        for error in errors {
-            let breadcrumb = Breadcrumb(level: .debug, category: "Startup")
-            breadcrumb.message = error
-            client.breadcrumbs.add(breadcrumb)
-        }
+        event.message = title
+
+        event.extra = errors.enumerated().reduce(into: [String: Any](), { (result, arg1) in
+            let (index, errorWithCaller) = arg1
+            let error = errorWithCaller.error
+            result["Error \(index + 1)"] = [
+                "Method": errorWithCaller.caller,
+                "Domain": error.domain,
+                "Code": error.code,
+                "Description": error.localizedDescription,
+                "User Info": error.userInfo.description
```

Comment:
This is fine and might actually help reading reports more easily, but just in case you didn't know, you are not limited to strings for those fields, you could have `error.userInfo` directly and sentry would show the right thing:

![Screen Shot 2020-02-14 at 12 50 21](https://user-images.githubusercontent.com/8739/74528980-95e01c00-4f28-11ea-9d92-6ee1e587234f.png)

Reply:
Hmm, I tried that and Sentry's SDK complained that it couldn't be serialised to JSON 🤔

Reply:
Ah yes, I think I saw some of that if any of the data types in userInfo wasn't serializable, `description` is probably safer 😄